crates/kubelet/src/config.rs

```diff
@@ -253,6 +260,14 @@ pub struct Opts {
         help = "The data path (logs, container images, etc) for krustlet storage. Defaults to $HOME/.krustlet"
     )]
     data_dir: Option<PathBuf>,
+
+    #[structopt(
+        long = "bootstrap-file",
+        env = "KRUSTLET_BOOTSTRAP_FILE",
+        help = "The path to the bootstrap config",
+        default_value = "/etc/kubernetes/bootstrap-kubelet.conf"
```

Comment:
Should this reference BOOTSTRAP_FILE? That way we're not maintaining the same string in two locations

Reply:
Let me see if that works. I've had problems using variables in these before